### ISSUE TITLE
new dataset loss problem

### ISSUE THREAD

**Issue by zperfet** [allenai/vampire#47]
hello, i use your model to my personal dataset, a 4 class  classification task silimar to IMDB. I followed all your steps but the performance is not that good. Even if i use all of my train data, the loss still decrease very slowly.  A simple MLP will increate acc from 0.25 to over 0.8 on my dataset, but vae can only increase acc from 0.25 to near 0.6, and take a much longer time.
I read your code for quite a long time,but i just can not figure out what is going wrong. can you give  me some advice about it ? Any advice will be appreciated.
![微信图片_20191119215903](https://user-images.githubusercontent.com/19884129/69152678-d2fa6500-0b17-11ea-9aeb-d5f84a49d6ca.png)

**Comment by kernelmachine** [comment]
Hi there! Sorry about the delay. I'm not sure I'll be able to work out the issue fully without digging into your data/problem space.

What is the quality of the topics that VAMPIRE learns? Low quality topics may suggest that VAMPIRE isn't learning well, and you could either tinker around with hyperparameters, or see if you can increase the size/quality of data.
